src/niftyone/io.py

```diff
@@ -10,11 +12,10 @@
 
 
 class VideoWriter:
-    """
-    A simple video streaming writer.
-    """
+    """A simple video streaming writer."""
 
-    def __init__(self, where: StrPath, fps: int):
+    def __init__(self, where: StrPath, fps: int) -> None:
+        """Initialize class."""
```

Comment:
Are docstrings required for dunder methods? Is there a way to disable this? It seems unnecessary and redundant to me.

Reply:
Yep - Added `D107` (init) and `D105` (all other dunder methods) to the ignore list, which will disable docstring checking for these.